docs/ar/week04/04.md

```diff
@@ -0,0 +1,15 @@
+---
+lang-ref: ch.04
+title: Week 4
```

Comment:
The title needs a translation.

Reply:
I translated the titles of both files and parts that I left untranslated in 04-1.md.
please check them and the config file and tell me if there's something wrong.